test/testutil/prepare_loggers.hpp

```diff
@@ -29,18 +34,23 @@ namespace testutil {
 groups:
   - name: libp2p
     sink: console
-    level: off
-  - name: libp2p_debug
-    level: trace
-)"));
+    level: info
+    children:
+      - name: testing
+# ----- End of logging system config addon -----
+)");
 
-      auto logging_system =
-          std::make_shared<soralog::LoggingSystem>(configurator);
+      auto logging_system = std::make_shared<soralog::LoggingSystem>(
+          std::make_shared<soralog::ConfiguratorFromYAML>(
+              // Original LibP2P logging config
+              std::make_shared<libp2p::log::Configurator>(),
+              // Additional logging config for testing
+              testing_log_config));
 
       auto r = logging_system->configure();
       if (r.has_error) {
         std::cerr << r.message << std::endl;
-        exit(EXIT_FAILURE);
+        throw std::runtime_error("Can't configure logger system");
```

Comment:
Why not pass r.message with the exception, by the way?

Reply:
yes, may be better runtime_error(fmt::format("Can't configure logger system: {}", r.message))